core/tests/org.fusesource.ide.camel.model.service.core.tests.integration/src/main/java/org/fusesource/ide/camel/model/service/core/tests/integration/core/model/GlobalDefinitionsIT.java

```diff
@@ -159,6 +158,18 @@ public void testCamelBeanPropertiesSpring() throws IOException, CoreException {
 		assertThat(bean3prop1.getAttribute(GlobalBeanEIP.PROP_NAME)).isEqualTo("description");
 		assertThat(bean3prop1.getAttribute(GlobalBeanEIP.PROP_VALUE)).isEqualTo("#1 account");
 		
+		CamelBean beanWithAllAttributes = (CamelBean)model1.getGlobalDefinitions().get("whateverWithAllFields");
+		checkAllCommonAttributesLoaded(beanWithAllAttributes);
+		assertThat(beanWithAllAttributes.getParameter(GlobalBeanEIP.PROP_FACTORY_BEAN)).isEqualTo("factoryMethod");
+	}
+
+	private void checkAllCommonAttributesLoaded(CamelBean beanWithAllAttributes) {
```

Comment:
I think because there are distinct differences between Global Bean properties for Spring and Blueprint, it would be good to call that out in the test by providing different methods. It's not as though we need a complete separate class, but it drives me crazy to see the Blueprint and Spring properties mixed.